supertokens_python/recipe/session/session_class.py

```diff
@@ -112,3 +119,77 @@ async def get_expiry(self, user_context: Union[Dict[str, Any], None] = None) ->
             raise_unauthorised_exception("Session does not exist anymore.")
 
         return session_info.expiry
+
+    async def assert_claims(
+        self,
+        claim_validators: List[SessionClaimValidator],
+        user_context: Union[Dict[str, Any], None] = None,
+    ) -> None:
+        if user_context is None:
+            user_context = {}
+
+        original_session_claim_payload_json = json.dumps(
+            self.get_access_token_payload()
+        )
+
+        new_access_token_payload = await update_claims_in_payload_if_needed(
+            claim_validators,
+            self.get_access_token_payload(),
```

Comment:
pass in user context to getAccessTokenPayload

Reply:
done.